src/protagonist/Engine/Ingest/AssetIngester.cs

```diff
@@ -70,14 +73,22 @@ public Task<IngestResult> Ingest(LegacyIngestEvent request, CancellationToken ca
     /// <returns>Result of ingest operations</returns>
     public async Task<IngestResult> Ingest(IngestAssetRequest request, CancellationToken cancellationToken = default)
     {
+        var asset = await assetRepository.GetAsset(request.Asset.Id);

```

Comment:
This could result in a cached read. Do we need control over what is loaded here (with `.Include()`)?